s3repo/model.py

```diff
@@ -340,6 +340,9 @@ def sync(self, permanent):
                     str(self.s3_settings['region']),
                 ]
 
+                if self.s3_settings.get('force_sync'):
+                    mkrepo_cmd.append('--force')
```

Comment:
Is it in effect only on startup? So, we'll decline pushing of a new malformed package?

Reply:
> Is it in effect only on startup? So, we'll decline pushing of a new malformed package?

As I see the use of this option:
1. we  launch the service with the flag and get the lists of malformed packages.
2. we solve problems manually
3. restart the service without the flag

About the effect. The option will be used until the service is restarted.

Reply:
Got it. Filed #26 about future enhancements we can do.